lib/core/src/lib/card-view/components/card-view-item-dispatcher/card-view-item-dispatcher.component.ts

```diff
@@ -57,6 +57,9 @@ export class CardViewItemDispatcherComponent implements OnChanges {
     @Input()
     multiValueSeparator: string = DEFAULT_SEPARATOR;
 
+    @Input()
+    displayLabelForChips: boolean = true;
```

Comment:
I'm not sure I understand, but how it is backward compatible this way? Didn't we say that we would keep the old behaviour as default?

Reply:
Done.